app/assets/stylesheets/assignments/calendar.scss

```diff
@@ -1,67 +1,19 @@
 #calendar {
-  border-collapse: collapse;
-  border-spacing: 0;
-  width: 98%;
-  height: 100%;
-  min-width: 800px;
-  margin: 0% 1%;
```

Comment:
I would still like some margin on the sides and bottom of the calendar. I don't like that the calendar ends where the color of the main-content ends on the sides and there should be a bit more space between the bottom and the calendar link.

![Screenshot 2021-02-17 131707](https://user-images.githubusercontent.com/26842354/108249575-3ca25e00-7123-11eb-87da-504ce100026f.jpg)